index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Jeremy Moore. Solutions.</title>
+  <link href="styles/base.css" type="text/css" rel="stylesheet" />
+  <link href="styles/layout.css" type="text/css" rel="stylesheet" />
+  <link href="styles/module.css" type="text/css" rel="stylesheet" />
+
+</head>
+<body>
+  <img src="images/hammer.png" id="hammer" alt="Hammer icon">
+  <nav>
+    <ul class="site-navbar">
+      <li class="tab" data-content"projects"><a href="#" >home</a></li>
```

Comment:
remove these <a> with href=#, you don't want an href anymore because we will handle the click with dom manipulate instead of navigating to a new page